docs/index.md

```diff
@@ -1,87 +1,34 @@
 # HTTP Adapter
 
-**This is the documentation for HTTP Adapter and it's software components.**
+**This is the documentation for the Httplug web client abstraction and the other PHP-HTTP components.**
 
-The HTTP Adapter abstracts from PHP HTTP clients that are based on [PSR-7](http://www.php-fig.org/psr/psr-7/).
+Httplug abstracts from HTTP clients written in PHP that are based on [PSR-7](http://www.php-fig.org/psr/psr-7/).
```

Comment:
I guess this wants to be that the abstraction is created based on existing PSR7 clients (which I wonder also if it's true, as the first PSR7 client was guzzle, and we started to work on the interface long before PSR7 was out)